pkg/block/indexheader/binary_reader.go

```diff
@@ -925,6 +925,16 @@ func (r *BinaryReader) postingsOffset(name string, values ...string) ([]index.Ra
 }
 
 func (r *BinaryReader) LookupSymbol(o uint32) (string, error) {
+	if r.indexVersion == index.FormatV1 {
```

Comment:
Before the change this came after the lookup! Is that problematic?

Reply:
Whoops I messed this up :(
Updated to the original order. @mhoffm-aiven  PTAL